pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -125,6 +123,11 @@ let
     && gccForLibs != null;
   gccForLibs_solib = getLib gccForLibs
     + optionalString (targetPlatform != hostPlatform) "/${targetPlatform.config}";
+  cc_solib =
+    if useGccForLibs then
+      gccForLibs_solib
+    else
+      getLib cc + optionalString (targetPlatform != hostPlatform) "/${targetPlatform.config}";
```

Comment:
Let's pick this up again, before nvcc begins to lag behind gcc releases again.

What happens here is that previously in the `useGccForLibs` branch there would be places where the wrapper references `gccForLbis_solib` and places where it references `cc.lib` via `cc_solib`.

This diff makes it so `useGccForLibs` changes the meaning of `cc_solib`: it is now the one lib that is propagated into the output.